LDRH_choice.py

```diff
@@ -460,16 +491,28 @@ def run_staircase(task, operation=None):
     except StopIteration:
         output={}
         output['score'] = 'StopIteration'
-        output['thisIncrement'] = handler.intensities[-1]
+        if not use_posterior_matching:
```

Comment:
We still want an output intensity from the handler, regardless of which handler we are using.